src/types.h

```diff
@@ -351,6 +354,10 @@ typedef union
 {
     syscall_info_t syscall_info;
     warning_info_t warning_info;
+    struct {
+        file_info_t scripts[4];
```

Comment:
Does this 4 correspond to MAX_INTERPRETERS?

Reply:
Yes and no. This is (1 script filename) + (MAX_INTERPRETERS - 1). The -1 is because we don't care about the last interpreter because that is the same as the executable.